core/time/shared/tz.rb

```diff
@@ -0,0 +1,39 @@
+describe :time_tz, shared: true do
+  describe "TZ env variable" do
+    before do
+      @tz = ENV["TZ"]
+
+      @get_time = @object
+    end
+
+    after do
+      ENV["TZ"] = @tz
+    end
+
+    it "returns time in timezone specified by TZ env variable" do
+      ENV["TZ"] = "Asia/Hong_Kong"
+      @get_time.call.zone.should == "HKT"
```

Comment:
Note: there is a `with_timezone` helper that can be helpful here

Reply:
@eregon Yeah, thanks.

I noticed this helper and actually it uses `TZ`. I just wanted to emphasize the effect of `TZ` variable explicitly in this specs regardless `with_timezone`'s implementation.